docs/trace/getting-started-jaeger/getting-started-jaeger.csproj

```diff
@@ -1,7 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Exporter.Console\OpenTelemetry.Exporter.Console.csproj" />
-    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Exporter.Jaeger\OpenTelemetry.Exporter.Jaeger.csproj" />
+    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Exporter.OpenTelemetryProtocol\OpenTelemetry.Exporter.OpenTelemetryProtocol.csproj" />
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Instrumentation.Http\OpenTelemetry.Instrumentation.Http.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Net.Http" Version="4.3.4" Condition="$(TargetFramework) == 'net472' or $(TargetFramework) == 'net48'" />
```

Comment:
@alanwest @utpilla This change piqued my interest. We don't specify targets explicitly. When I build this project locally I get output for net7.0, net6.0, net462, net47, net471, net472, & net48. It is interesting that net462, net47, & net471 don't also need this reference 🤔 But it raises a question: Should we specify targets on these docs projects?

Reply:
Disregard! I found the targets defined here: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/Directory.Build.props#L8-L10